test/integration/app/users/update-language.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Users } from "@app"
+import { InvalidLanguageError } from "@domain/errors"
+
+import { getUserIdByTestUserIndex } from "test/helpers"
+
+let userId0: UserId
+
+beforeAll(async () => {
+  userId0 = await getUserIdByTestUserIndex(0)
```

Comment:
maybe we can create the user also before?

I'm trying for every test I'm working on to add this at the beginning of the test, so that we could eventually whip out the db between every tests for more consistency, for the tests that don't depend on global consistency with lnd/bitcoin